lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -246,9 +246,20 @@ export default function BasicLTILaunchApp() {
     }
   }, [authToken, canvas.speedGrader, contentReady]);
 
+  // Report a submission only after the first qualifying annotation activity
+  // (create or update an annotation or reply)
   useEffect(() => {
-    reportSubmission();
-  }, [reportSubmission]);
+    const unsubscribe = () =>
+      clientRPC.off('annotationActivity', onAnnotationActivity);
+
+    function onAnnotationActivity() {
+      reportSubmission().then(unsubscribe);
```

Comment:
I initially wrote a comment here about error handling, but I see that `reportSubmission` does catch errors itself.

Reply:
OK, yeah, I saw your first comment and was like "but it does :)"